plugins/mergePaths.js

```diff
@@ -17,7 +17,7 @@ exports.description = 'merges multiple paths in one if possible';
 exports.fn = (root, params) => {
   const {
     force = false,
-    floatPrecision,
+    floatPrecision = 3,
```

Comment:
If I don't do this, whenever I set an exact value for an argument it leaks out, despite my plugin having a default set.